src/rpc/rawtransaction.cpp

```diff
@@ -132,7 +132,7 @@ static RPCHelpMan getrawtransaction()
                                      {
                                          {RPCResult::Type::STR, "asm", "the asm"},
                                          {RPCResult::Type::STR, "hex", "the hex"},
-                                         {RPCResult::Type::NUM, "reqSigs", "The required sigs"},
+                                         {RPCResult::Type::NUM, "reqSigs", /* optional */ true, "(DEPRECATED, returned only if config option -deprecatedrpc=reqSigs is passed) Number of required signatures (only present for bare multisig"},
```

Comment:
I don't think it's technically true anymore in the current PR that this is only present for bare multisig (here and elsewhere).